data/transport missions.txt

```diff
@@ -1951,3 +1951,369 @@ ship "Science Drone"
+				accept
+
+
+
+mission "Lost Boy 2"
+	landing
+	name `Rescue Tod`
+	description `Locate the <npc> and board it to rescue Tod, then bring him to <destination> so that he may reunite with his mother.`
+	destination Moonshake
+	to offer
+		has "Lost Boy 1: done"
+	
+	on offer
+		conversation
+			`The air in the <origin> spaceport smells of tobacco and tastes of the salt from the world's vast oceans. You receive sideways looks from rough-looking pirates, but none make a move to cause any trouble. As lawless as pirates can be, they still adhere to a form of honor code when in a planet's main spaceport. At least generally.`
+			`	You ask a bartender for directions to the local slave traders, as that is the likely place that Tod would be if he was taken here. He points you to a slave trader by the name of Cygnet Brig who "runs the slave trade 'round these parts."`
+			`	Cygnet owns a large warehouse building located on the edge of the spaceport island where he keeps all his slaves packed together in groups of three in small holding cells. Some slaves are rattling the bars on their cell or screaming in protest, but many have seemed to accept their fate. Cygnet, a tall, thin, mustachioed man, notices you gazing at the cells. "Can't keep 'em too packed together, oth'wise they's get each otha sick," he remarks in a thick accent of unknown origin to you. "What can I get ya?"`
+			choice
+				`	"I'm here to buy a slave."`
+					goto description
+				`	"Do you have a boy by the name of Tod Copper?"`
+			
+			`	Cygnet strokes his mustache. "Tod Copper. Tod Copper. Tell you what, that name doesn' ring a bell. Heck, I don' even keep track of names 'less I get someone impor'ant. If ya lookin' for someone in particular to buy, can I get a description?"`
+			
+			label description
+			`	You provide Cygnet with the description of Tod that his mother gave you, then explain to him that Tod would have been taken from a ship bound for Placer.`
+			`	"Oh yeah, that bugger! That little rebel was causin' me trouble so I sold him off soon as I could. If ya looking for him, then you're gonna need to look for a guy by the name of <npc>. He bought your boy Tod 'bout four days ago. He might be hangin' around a system nearby."`
+			`	You thank Cygnet, who has been surprisingly polite for a slave trader, for the information. "If ya ever need a slave, just come talk to me," he remarks as you walk back to your ship. "I've got the cheapest slaves this side of Sol."`
+				accept
+	
+	npc board
+		personality staying nemesis target plunders
+		government "Bounty"
+		system
+			distance 2 4
+		ship "Vanguard (Particle)" "Lord Ligonier"
+		conversation
+			`When you breach the ship's hull, you find the crew of the <npc> more concerned with containing the fires caused by your weapons than they are you walking onto their ship. Almost every one of the crew members is a teenager. One looks at you and the laser pistol in your hand with mortal fear in his eyes.`
```

Comment:
The first sentence's grammar doesn't work; it should be "...than they are with you walking onto their ship" or "than with you walking onto their ship."